.github/workflows/test-dotnet.yaml

```diff
@@ -13,9 +13,9 @@ jobs:
       - name: 🏗 Build prqlc-clib
         run: cargo build --package prqlc-clib --release 
       - name: 🔢 Set version
-        run: ./.github/workflows/scripts/set_version.sh
-      - name: 🐜 Debug CI/CD (version)
-        run: echo $version
+        run: |
+            $version = cargo metadata --format-version=1 --no-deps | jq --raw-output '.packages[] | select(.name == "prql-compiler") | .version'
+            echo "version=$version" >> $env:GITHUB_ENV
```

Comment:
I think this is not a pwsh syntax.
I suppose you should just set `shell: bash` to fix that.

Reply:
It should work on PowerShell (at least it's working on my computer with PowerShell Core 7.3.9)

Reply:
Finally, found a solution for this. Not sure why but my issue was with the jq query :-D ... moving to next blocker.

Reply:
Please use the original script.
If the same process is written in multiple places, it will be a problem when updating later.

Reply:
Hmmm, not sure that I can run a bash script on Windows. I will try.

Reply:
It can be executed. It is already used in the following places:
https://github.com/PRQL/prql/blob/56bdbe4bd23ca3c2b625c0559d01ecdde078ecdf/.github/actions/build-prqlc/action.yaml#L25-L26